gui/GameConqueror.py

```diff
@@ -345,7 +345,6 @@ def __init__(self):
     # GUI callbacks
     
     # Memory editor
-
```

Comment:
Stop messing with white lines, there are a bunch of these (I know you didn't do it on purpose, it's the result of 2 commits).